web/tests/list_widget.test.js

```diff
@@ -561,66 +561,66 @@ run_test("clear_event_handlers", () => {
-
-    blueslip.expect("error", "Filterer and predicate are mutually exclusive.");
-    ListWidget.create($container, list, {
-        filter: {
-            filterer: /* istanbul ignore next */ () => true,
-            predicate: /* istanbul ignore next */ () => true,
-        },
-        $simplebar_container: $scroll_container,
-        get_item() {},
-    });
-    blueslip.reset();
-
-    blueslip.expect("error", "Filter filterer is not a function (or missing).");
-    ListWidget.create($container, list, {
-        filter: {},
-        $simplebar_container: $scroll_container,
-        get_item() {},
-    });
-    blueslip.reset();
-
-    blueslip.expect("error", "List item is not a string");
-    ListWidget.create($container, list, {
-        modifier: () => 999,
-        $simplebar_container: $scroll_container,
-        get_item: (item) => item,
-    });
-    blueslip.reset();
-});
+// run_test("errors", () => {
+//     // We don't care about actual data for this test.
+//     const list = ["stub"];
+//     const $container = make_container();
+//     const $scroll_container = make_scroll_container();
+
+//     blueslip.expect("error", "Need opts to create widget.");
+//     ListWidget.create($container, list);
+//     blueslip.reset();
+
+//     blueslip.expect("error", "$simplebar_container is missing.");
+//     ListWidget.create($container, list, {
+//         modifier: "hello world",
+//     });
+//     blueslip.reset();
+
+//     blueslip.expect("error", "get_item should be a function");
+//     ListWidget.create($container, list, {
+//         get_item: "not a function",
+//         $simplebar_container: $scroll_container,
+//     });
+//     blueslip.reset();
+
+//     blueslip.expect("error", "Filter predicate is not a function.");
+//     ListWidget.create($container, list, {
+//         filter: {
+//             predicate: "wrong type",
+//         },
+//         $simplebar_container: $scroll_container,
+//         get_item() {},
+//     });
+//     blueslip.reset();
+
+//     blueslip.expect("error", "Filterer and predicate are mutually exclusive.");
+//     ListWidget.create($container, list, {
+//         filter: {
+//             filterer: /* istanbul ignore next */ () => true,
+//             predicate: /* istanbul ignore next */ () => true,
+//         },
+//         $simplebar_container: $scroll_container,
+//         get_item() {},
+//     });
+//     blueslip.reset();
+
+//     blueslip.expect("error", "Filter filterer is not a function (or missing).");
+//     ListWidget.create($container, list, {
+//         filter: {},
+//         $simplebar_container: $scroll_container,
+//         get_item() {},
+//     });
+//     blueslip.reset();
+
+//     blueslip.expect("error", "List item is not a string");
+//     ListWidget.create($container, list, {
+//         modifier: () => 999,
+//         $simplebar_container: $scroll_container,
+//         get_item: (item) => item,
+//     });
+//     blueslip.reset();
+// });
```

Comment:
Should we just remove the test cases?

Reply:
Yeah I think it makes sense to remove them now that we don't have over-defensive code to raise those errors 👍